master/getting-started/bare-metal/bare-metal.md

```diff
@@ -476,31 +477,83 @@ outside the cluster:
    normal host endpoint policy).  In other words, if a host endpoint is defined
    but has no pre-DNAT policies that explicitly allow or deny a particular
    incoming packet, that packet is allowed to continue on its way, and will
-   then be accepted or dropped according to workload policy (if it is going to
-   a local workload) or to normal host endpoint policy (if not).
+   then be accepted or dropped according to normal host endpoint policy applied on
+   forward traffic, or a workload policy.
+
+## Policy on forwarded traffic
+If `applyOnForward` is `false`, host endpoint policy applies to traffic to/from
+ local processes only. 
+
+If `applyOnForward` is `true`, host endpoint policy applies to forwarded traffic:
+- Incoming traffic from a host endpoint to a local workload (container/pod/VM).
+- Outgoing traffic from a local workload to a host endpoint.
+- Traffic from one host endpoint that is forwarded to another host endpoint.
+
+By default, `applyOnForward` is `false`. 
+
+Untracked policies and pre-DNAT policies must have `applyOnForward` set to `true`
+because they apply to all forwarded traffic.
+
+Forwarded traffic is allowed by default. In other words, if a host endpoint is
+configured, but there are no rules that explicitly allow or deny a particular
+forwarding flow (in any of policies with applyOnForward true that apply to that
+host endpoint), that flow is still allowed. This is different from how Calico
+treats traffic to or from a local process: traffic to or from a local process is
+denied by default, if a host endpoint is configured but there is no applicable
+policy that explicitly allows that traffic.
+
+> **NOTE**: Calico's handling of host endpoint policy has changed, since before
+> Calico v2.7.0, in two ways:
+> - It will not apply at all to forwarded traffic, by default. If you have existing
+> policy and you want it to apply to forwarded traffic, you need to add `applyOnForward: true`
+> to existing policy.
+> - Even with `applyOnForward: true`, the treatment is not quite the same in
+> Calico v2.7.0 as in previous releases, because - once a host endpoint is configured -
```

Comment:
" - " --> "–"
(Use the actual em dash character (which I've pasted above) and no spaces around it.